go.mod

```diff
@@ -22,8 +22,8 @@ require (
 	github.com/cilium/ebpf v0.3.0
 	github.com/cilium/ipam v0.0.0-20201106170308-4184bc4bf9d6
 	github.com/cilium/proxy v0.0.0-20201222161048-8bceb17eb027
-	github.com/containernetworking/cni v0.7.1
-	github.com/containernetworking/plugins v0.8.2
+	github.com/containernetworking/cni v0.8.0
+	github.com/containernetworking/plugins v0.9.0
```

Comment:
We might need to split this one into a different commit and put it together with the changes that are required in here as well:
https://github.com/cilium/cilium/blob/0a444b7aa046d7b55f0c7d4cb4ef92bac6247b03/contrib/packaging/docker/Dockerfile.runtime#L53-L57

Reply:
Good point. For now I've dropped the `github.com/containernetworking/*` and `github.com/ginkgo/*` bumps from this PR and will send separate PRs for these after local testing.